.github/files/fips/requirements.yml

```diff
@@ -1,6 +1,7 @@
 ---
 roles:
   - name: geerlingguy.postgresql
-    version: 2.2.1
+    src: https://github.com/fao89/ansible-role-postgresql.git
+    version: fips_scram_sha256
```

Comment:
You should also update vagrant/boxes.d/ (all the fips boxes specifically) to use this requirements file.